src/lib/components/actions.svelte

```diff
@@ -50,6 +50,17 @@
 		event.preventDefault();
 	};
 
+	const getSvgEl = () => {
+		const svgEl: HTMLElement = document
+			.querySelector('#container svg')
+			.cloneNode(true) as HTMLElement;
+		const styleEl = document.createElement('style');
+		styleEl.innerText =
+			'@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css");';
```

Comment:
Would you mind extract the version as a variable so it can be changed more easily?

Reply:
Sure thing. I am not super familiar with SvelteKit. Is there a place to set the variable so it can be referenced both here and [app.html](https://github.com/mermaid-js/mermaid-live-editor/blob/master/src/app.html#L18)?

Reply:
See this issue comment https://github.com/sveltejs/kit/issues/3091#issuecomment-1112589090 on custom templating in `app.html`.

Also, let's just keep the import using a `<link>` tag inside the `app.html` as putting the import in the CSS could possibly cause it to import every single time the user types into the textbox of the editor.

Reply:
I updated the code to read the font-awesome cdn url from `document.head` so it is no longer hard coded. Seems like a less invasive change then extracting it as a shared `const` and adding `transformPage` to the hook.